example/themes/original/src/components/Cart/index.tsx

```diff
@@ -353,6 +355,24 @@ const CartUI = (props: any) => {
                 </OText>
               </OSTable>
             </OSTotal>
+            {[3, 4].includes(orderState?.options?.type) && (
```

Comment:
please don't use this validation because is hardcoded, create a constant for this